ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -821,3 +837,105 @@ def test_clear_job_queue(self, mock_get_drac_client):
+                    enabled_management_interfaces=['idrac-redfish'],
+                    enabled_inspect_interfaces=['idrac-redfish'])
+        self.node = obj_utils.create_test_node(
+            self.context, driver='idrac',
+            driver_info=DRAC_REDFISH_INFO_DICT)
+
+    def test_get_properties(self, mock_redfish_utils):
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=True) as task:
+            properties = task.driver.get_properties()
+            for prop in redfish_utils.COMMON_PROPERTIES:
+                self.assertIn(prop, properties)
+
+    @mock.patch.object(oem_manager, 'delete_jobs', autospec=True)
+    def test_clear_job_queue(self, mock_clear_job_queue, mock_redfish_utils):
+        mock_system = mock_redfish_utils.get_system.return_value
+        mock_manager = mock.MagicMock()
+        mock_system.managers = [mock_manager]
+        mock_manager_oem = mock_manager.get_oem_extension.return_value
+        mock_manager_oem = mock_manager.get_oem_extension.return_value
+        response_data = {
+            'status_code': 200
+        }
+
+        mock_manager_oem.delete_jobs.return_value = test_utils.DictToObj(
+                                                                response_data)
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            task.node.driver_info['redfish_address'] = '1.2.3.4'
+            return_value = task.driver.management.clear_job_queue(task)
+            self.assertIsNone(return_value)
+            mock_manager_oem.delete_jobs.assert_called_once_with(task,
+                                                    job_ids=['JID_CLEARALL'])
+
+    @mock.patch.object(oem_manager, 'reset_idrac', autospec=True)
+    @mock.patch.object(drac_utils, 'wait_for_host', autospec=True)
+    @mock.patch.object(drac_utils, 'wait_until_idrac_is_ready', autospec=True)
+    def test_reset_idrac(self, 
+                        mock_reset_idrac, 
+                        mock_wait_for_host,
+                        mock_wait_until_idrac_ready,
+                        mock_redfish_utils):
+        mock_system = mock_redfish_utils.get_system.return_value
+        mock_manager = mock.MagicMock()
+        mock_system.managers = [mock_manager]
+        mock_manager_oem = mock_manager.get_oem_extension.return_value
+        response_data = {
+            'status_code': 200
+        }
+
+        mock_manager_oem.reset_idrac.return_value = test_utils.DictToObj(
+                                                                response_data)
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            task.node.driver_info['redfish_address'] = '1.2.3.4'
+            return_value = task.driver.management.reset_idrac(task)
+            self.assertIsNone(return_value)
```

Comment:
we could probably cut out the line above.  The doc for reset_idrac in python-dracclient says that it returns True on success, so validating that it returned None is probably not the best thing to do.